javascript/javascript1/week3/series.js

```diff
@@ -0,0 +1,50 @@
+const seriesDurations = [
+    {
+      title: "Monster",
+      days: 1,
+      hours: 5,
+      minutes: 36,
+    },
+    {
+      title: "Gilmore Girls",
+      days: 5,
+      hours: 21,
+      minutes: 48,
+    },
+    {
+      title: "Samurai Shamploo",
+      days: 6,
+      hours: 8,
+      minutes: 12,
+    },
+  ];
+  
+  const lifespan = 80; 
+  const lifespanMinutes = lifespan * 365 * 24 * 60; 
+  
+  function getPercentage() {
```

Comment:
As you start working on more complex projects and with other people, it will quickly become apparent that naming is of immense importance!  Here I think people would expect to get a number back when it is called `getPercentage`. Can you come up with a name that better describes what the function does? 😄